public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -44,5 +47,5 @@ enum class PurchasesErrorCode(val description: String) {
     IneligibleError("The User is ineligible for that action."),
     InsufficientPermissionsError("App does not have sufficient permissions to make purchases."),
     PaymentPendingError("The payment is pending."),
-    InvalidSubscriberAttributesError("One or more of the attributes sent could not be saved.")
+    InvalidSubscriberAttributesError(AttributionStrings.INVALID_SUBSCRIBER_ATTRIBUTES)
```

Comment:
another case where we could improve consistency: 
`INVALID_CREDENTIALS_ERROR`
`INVALID_SUBSCRIBER_ATTRIBUTES`
we could have either both have the suffix `ERROR` or neither of them